doc/api/net.md

```diff
@@ -100,47 +169,65 @@ deprecated: v0.9.7
 
     Stability: 0 - Deprecated: Use [`server.getConnections()`][] instead.
 
-The number of concurrent connections on the server.
+A property indicating the number of concurrent connections on the server.
 
-This becomes `null` when sending a socket to a child with
+The value is `null` when a socket is sent to a child using
 [`child_process.fork()`][]. To poll forks and get current number of active
-connections use asynchronous `server.getConnections` instead.
+connections use the `server.getConnections()` method instead.
 
 ### server.getConnections(callback)
 <!-- YAML
 added: v0.9.7
 -->
 
-Asynchronously get the number of concurrent connections on the server. Works
-when sockets were sent to forks.
+* `callback` {Function}
+
+The `server.getConnections()` method asynchronously returns the number of
+concurrent connections on the server, including those held by child processes
+opened using [`child_process.fork()`][].
 
-Callback should take two arguments `err` and `count`.
+The `callback` function is invoked with two arguments:
+
+* `err` {Error} `null` if the operation was successful or `Error` if an
+  error occurred.
+* `count` {number} The number of connections.
+
+```js
+server.getConnections((err, count) => {
+  if (err) throw err;
+  console.log(`There are ${count} active connections for this server.`);
+});
+```
 
 ### server.listen(handle[, backlog][, callback])
 <!-- YAML
 added: v0.5.10
 -->
 
 * `handle` {Object}
-* `backlog` {Number}
+* `backlog` {Number} Defaults to `511`.
 * `callback` {Function}
 
-The `handle` object can be set to either a server or socket (anything
-with an underlying `_handle` member), or a `{fd: <n>}` object.
+The `server.listen()` method instructs the server to begin accepting connections
+on the given `handle`. The `handle` object may either be any object with an
+underlying `_handle` member (such as a `net.Socket` or another `net.Server`),
+or an object with a `fd` property whose value is a numeric file descriptor
+(e.g. `{fd: <n>}`).
```

Comment:
but this will only work if the fd refers to a socket, right? or maybe a pipe? but definitely not a file.